src/sql/platform/connection/common/connectionStatusManager.ts

```diff
@@ -225,6 +225,10 @@ export class ConnectionStatusManager {
 		return !!(uri && uri.startsWith(Utils.uriPrefixes.default));
 	}
 
+	public isEditorTypeUri(uri: string): boolean {
+		return !!(uri && (uri.startsWith('untitled:') || uri.startsWith('file:')));
```

Comment:
Use [Schemas](https://github.com/microsoft/azuredatastudio/blob/main/src/vs/base/common/network.ts#L9) for the schema names